NFive.SDK.Core.sln

```diff
@@ -1,24 +1,22 @@
+﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2035
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.105
```

Comment:
Its worth defaulting to VS22 but it would be better if only these two lines are changed in the SLN files, so that the projects keep the same GUIDs.